sharepoint/sharepoint-ps/sharepoint-online/New-SPOListFileVersionBatchDeleteJob.md

```diff
@@ -16,7 +16,7 @@ manager: seanmc
 ## SYNOPSIS
 
 > [!NOTE]
-> This feature is part of the Version history controls Preview. If your tenant is not part of the Preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
+> This feature is part of the version history controls preview. If your tenant is not part of the preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
 
 Starts a file version batch trim job targeting all document libraries in a site collection.
```

Comment:
@msjennywu Isn't this cmdlet for library scoped version trimming ?

Queue a job to trim versions from a document library. 

Caution: Versions trimmed using this command will be permanently deleted and cannot be recovered from the recycle bin.

Reply:
You are right!

Added "Queue a job to trim versions from a document library. " in synopsis, added all in description.

Reply:
@rekamath should it be "Queues"? (e.g. it was "Starts..." before)